beta/do-app-platform-op-cli/README.md

```diff
@@ -0,0 +1,146 @@
+
+```sh
+op item edit "scimsession file" --vault "op-scim" base64_encoded=$(op read "op://op-scim/scimsession file/scimsession" | base64 | tr -d "\n")
+```
+
+> **Note**
+>
+> If you used a different vault or item name for your SCIM bridge credentials, replace `op-scim` and `scimsession file` with the respective name(s) you chose.
+
+## Deploy 1Password SCIM bridge to App Platform
+
+Stream the app spec template from this repository, use [`op inject`](https://developer.1password.com/docs/cli/reference/commands/inject) to load in the Base64-encoded `scimesssion` credentials from your 1Password account, then pipe the output into `doctl` to deploy 1Password SCIM bridge.
+
+For example, with `curl`:
+
+```sh
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/beta/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait
+```
+
+1Password SCIM bridge deploys with a live URL output to the terminal (found under the `Default Ingress` column). Use your bearer token with the URL to test the connection to 1Password. For example:
+
+```sh
+curl --header "Authorization: Bearer $(op read op://${VAULT:-op-scim}/${ITEM:-"bearer token"}/credential)" https://op-scim-bridge-example.ondigitalocean.app/Users
+```
+
+You can also access your SCIM bridge by visting the URL in your web browser. Sign in with the bearer token saved in your 1Password account.
+
+## Appendix
+
+### Supply custom vault and item names
+
+If you chose your own name for the vault and items where you saved your SCIM bridge credentials, you can override the defaults using the `VAULT` and `ITEM` variables in the secret references. For example:
+
+```sh
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/beta/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
+```
+
+### Update 1Password SCIM bridge
+
+The latest version of 1Password SCIM bridge is posted on our [Release Notes](https://app-updates.agilebits.com/product_history/SCIM) website, where you can find details about the latest changes. The most recent version should also be pinned in [`op-scim-bridge.yaml`](./op-scim-bridge.yaml), so you can update using the same command as above with the `--upsert` parameter:
+
+```sh
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/beta/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
+```
+
+### Propose the app spec
+
+You can optionally propose the raw app spec template to verify the cost before deploying to DigitalOcean:
+
+```sh
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/beta/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
+```
+
+<!--
+TODO: Add instructions for vertical scaling
+
+### Scaling
+
+ -->
+
+<!-- TODO: Allow Workspace credentials to be loaded -->
```

Comment:
I understand this is a beta project, but I'd prefer to see these represented in the Readme as a more formal todo list.

EG:
```md

## TODO
 
- [ ] Add instructions for vertical scaling
- [ ] Enable the automatic loading of workspace credentials
```

Reply:
Fair enough. Noted and amended. Thanks for the review!